package.json

```diff
@@ -12,6 +12,10 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
```

Comment:
Since we would have to ship the node modules anyway, it would be ok to put it in the extension directory. Also will avoid the NPM setups needed otherwise.